state/converters.go

```diff
@@ -151,9 +151,8 @@ func (s *State) convertToProcessTransactionResponse(txs []types.Transaction, res
 		result.CallTrace = *callTrace
 		result.EffectiveGasPrice = response.EffectiveGasPrice
 		result.EffectivePercentage = response.EffectivePercentage
-		result.Tx = txs[i]
 
-		_, err = DecodeTx(common.Bytes2Hex(response.GetRlpTx()))
+		tx, err := DecodeTx(common.Bytes2Hex(response.GetRlpTx()))
```

Comment:
Is there any way to use the txs received from the executor?? Maybe it is better to rely on the executor to decode txs

Reply:
Yes, this is the TX we later store in the trusted state. The executor returns it in rlp format.